ts_src/index.ts

```diff
@@ -18,4 +20,6 @@ export {
   FeeProvider,
   FixedFeeProvider,
   SizeBasedFeeProvider,
+  Balance,
+  Utxo,
```

Comment:
I couldn't find balance.ts and utxo.ts. These are neccessary?

Reply:
https://github.com/chaintope/tapyrusjs-wallet/blob/main/ts_src/utxo.ts
https://github.com/chaintope/tapyrusjs-wallet/blob/main/ts_src/balance.ts